compiler/bsc/rescript_compiler_main.ml

```diff
@@ -41,6 +69,7 @@ let process_file sourcefile ?kind ppf =
      properly
   *)
   setup_outcome_printer ();
+  Error_message_utils_support.setup ();
```

Comment:
I think `process_file` is the appropriate place to run this, but maybe there's a better place, idk.